src/test/java/org/cactoos/scalar/package-info.java

```diff
@@ -25,6 +25,10 @@
 /**
  * Scalars, tests.
  *
+ * @todo #1350:30min Continue replacing usage of MatcherAssert.assertThat with
```

Comment:
@iiylll the bug you are fixing is not 1350 but 1343 I believe, no?

Reply:
Yes, but how could I add another TODO if I didn't replace all MatcherAssert.assertThat in the project?

Reply:
@iiylll the principle is the following: when you add a todo, you start it with `#xxx:30min` with `xxx` being the ticket currently being solved. so that when someone solve your todo, then know where does it come from.

So, currently you are solving the ticket #1343, then because we consider that there is more work to finish the ticket you are doing, you remove the previous todo and you need to add a new todo starting with `#1343:30min` so that someone else can continue the work until there are no `MatcherAssert.assertThat` in the project.

Reply:
okay, I fixed that, could you check it, please?